Source/i_system.c

```diff
@@ -90,9 +95,24 @@ int I_GetTime_RealTime(void)
 int realtic_clock_rate = 100;
 int clock_rate;
 
+static Uint32 GetTimeMS_Scaled(void)
+{
+  Uint32 time;
+
+  if (clock_rate == 100)
+    time = SDL_GetTicks();
+  else
+    time = SDL_GetTicks() * clock_rate / 100;
+
+  if (basetime == 0)
+    basetime = time;
+
+  return time - basetime;
```

Comment:
I believe the `clock_rate / 100` factor has to be applied to the `time - basetime` difference? Imagine we start with `clock_rate == 100` and then immediately switch to `clock_rate = 10` in the next tic. Then we would substract a `time` value that is smaller than `basetime`, or not?

Reply:
`basetime` adjusted in `I_InitTimer()`